api/v1alpha1/schedule_type_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Chaos Mesh Authors.
```

Comment:
s/2019/2021/g  ?

Reply:
oops, that is copy&pasted. I may do a global fix on this.